### DISCUSSION TITLE
Vaultwarden - Disable registration of new users

### DISCUSSION THREAD

**Discussion by rul3zz** [tteck/Proxmox#1591] [Help Needed] [answered]
How can I disable the registration of a new user? (Regular installation, not Alpine).
I've tried searching on the Admin page, but I couldn't find any options.

**Comment by tteck** [comment, marked as answer]
In the Vaultwarden Admin Panel, under General Settings

![Screenshot 2023-07-05 5 30 39 AM](https://github.com/tteck/Proxmox/assets/21342670/d20418d0-f438-4d97-bad6-4bbff72395ca)

**Comment by rul3zz** [comment]
Oh my goodness, I must be blind. I must have looked several times without seeing it! Thank you for your patience! And most importantly, thank you for making the installation so straightforward.

**Reply by OsipovOV** [reply]
I removed the allow new signups checkbox and checked that a new user cannot register. But after I disable the admin panel (commenting out the line admin_token..... in /opt/vaultwarden/config.json and /opt/vaultwarden/.env) the new user can register again. Is this like this for everyone? How to simultaneously disable new signups and admin panel?
